src/documentation/ApiDocsAccordion.tsx

```diff
@@ -17,16 +18,47 @@ interface ApiDocsAccordionProps {
 }
 
 const ApiDocsAccordion = ({ docs }: ApiDocsAccordionProps) => {
+  const modules = useMemo(
+    () => sortBy(Object.values(docs), (m) => m.fullName),
+    [docs]
+  );
+  const [expandedIndex, setExpandedIndex] = useState<
+    ExpandedIndex | undefined
+  >();
+  const scrollTarget = useRef<string | undefined>();
+  useEffect(() => {
+    const listener = (event: Event) => {
+      const id = (event as CustomEvent).detail.id;
+      const module = id.slice(0, id.lastIndexOf("."));
+      const index = modules.findIndex((m) => m.fullName === module);
+      scrollTarget.current = id;
+      setExpandedIndex(index);
+    };
+    document.addEventListener("cm/openDocs", listener);
+    return () => {
+      document.removeEventListener("cm/openDocs", listener);
+    };
+  }, [modules]);
+  useEffect(() => {
+    if (scrollTarget.current) {
+      const elt = document.getElementById(scrollTarget.current);
+      console.log("Scrolling", elt);
+      scrollTarget.current = undefined;
+      elt?.scrollIntoView();
+    }
```

Comment:
This doesn't scroll unless the index changed, which isn't what we want.

It also feels like I'm missing the simple way to do this! We need to know the render is done in the case where the index does change, hence the second effect but it's not very lovely.